packages/docs/website/pages/en/index.js

```diff
@@ -60,7 +60,6 @@ class HomeSplash extends React.Component {
 
     return (
       <SplashContainer>
-        <Logo img_src={`${baseUrl}img/docusaurus.svg`} />
```

Comment:
maybe add DINA logo? it's in `packages/ui/src/apps/collectionsUi/viewModules/home/logo.png`

Reply:
Will try and see what it looks like :)